node/src/BaseClient.ts

```diff
@@ -2962,6 +2963,51 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Stores a specified range of elements from the sorted set at `source`, into a new
+     * sorted set at `destination`. If `destination` doesn't exist, a new sorted
+     * set is created; if it exists, it's overwritten.
+     *
+     * See https://valkey.io/commands/zrangestore/ for more details.
+     *
```

Comment:
Do we need to add since 6.2.0? Other clients don't have the since tag.

Reply:
We need it there and other clients too